src/foam/comics/DAOControllerView.js

```diff
@@ -185,7 +190,10 @@ foam.CLASS({
                   this
                     .start()
                       .add(self.cls.PREDICATE.clone().copyFrom({
-                        view: { class: 'foam.u2.view.SimpleSearch' }
+                        view: { 
+                          class: 'foam.u2.view.SimpleSearch',
+                          searchValue: self.memento && self.memento.paramsObj.search || null
```

Comment:
Is null treated different than undefined?

Reply:
no

Reply:
Then you can remove the '|| null'.

Reply:
do you mean `searchValue: self.memento.paramsObj.search`?
if so, what if self.memento is null?

Reply:
Just:
self.memento && self.memento.paramsObj.search
leave off the '|| null', which doesn't do anything

Reply:
won't that return boolean value instead of self.memento.paramsObj.search?